concepticondata/concepticon.tsv

```diff
@@ -2303,8 +2303,8 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2302	CIRCULAR (ROUND IN TWO DIMENSIONS)	Spatial relations	Having a circular shape.	Property	
 2303	SPHERICAL (ROUND IN THREE DIMENSIONS)	Spatial relations	Having a spherical shape.	Property	
 2304	THEE (OBLIQUE CASE OF YOU)	Kinship	Oblique cases of the second person singular pronoun.	Other	
-2305	THIN (OF HAIR AND THREAD)	Spatial relations	Being very narrow (of long objects, like &quot;hairs&quot; or &quot;wires&quot;).	Property	
-2306	THIN (OF LEAF AND CLOTH)	Spatial relations	Being very narrow (of flat objects, like &quot;leaves&quot; or &quot;cloth).	Property	
+2305	THIN (OF HAIR AND THREAD)	Spatial relations	Being very narrow (of long objects, like hairs or wires).	Property	
+2306	THIN (OF LEAF AND CLOTH)	Spatial relations	Being very narrow (of flat objects, like leaves or cloth).	Property	
```

Comment:
Here the 'very' is far more acceptable (see comment for THIN (OF SHAPE OF OBJECT) above)